### DISCUSSION TITLE
Chrome (possibly other browsers too) field autocomplete for search in nav bar causes reload

### DISCUSSION THREAD

**Discussion by jlvallelonga** [AllYourBot/hostedgpt#587] [General]
[Screencast from 12-16-2024 10:55:08 AM.webm](https://github.com/user-attachments/assets/1df3c4f6-3a9b-4fa4-8e92-ec07ea883142)

click on the autocomplete option causes the field to reload. maybe just needs a preventDefault somewhere?

also, not entirely related, but search doesn't search the title of the chat. pretty small issue, but might be useful if users have a lot of chats.

I'm running Chrome Version 130.0.6723.116 (Official Build) (64-bit) on Ubuntu 22.04.5 LTS

**Comment by keithschacht** [comment]
@jlvallelonga thanks for flagging this autocomplete issue. But it should definitely be searching the title of the chat. Can you try again? It's working for me and we have a test case for that one.

@mattlindsey want to take a look at this autocomplete one? Justin's idea of preventDefault is a good one. I don't quite understand what keystrokes the browser autocomplete are triggering which causes it to clear/refresh.

**Comment by jlvallelonga** [comment]
looks like the title search thing isn't an issue anymore
